CHANGELOG.md

```diff
@@ -2,11 +2,16 @@
 
 ## v1.0.0-beta3 (unreleased)
 
+* Add `task.MustRegister` convenience function which fails fast by panic-ing
+  Note that this should only be used during app initialization, and is provided
+  to avoid repetetive error checking for services which register many tasks.
+
 ## v1.0.0-beta2 (09 Mar 2017)
 
 * [Breaking] Remove `ulog.Logger` interface and expose `*zap.Logger` directly.
 * [Breaking] Rename config and module from `modules.rpc` to `modules.yarpc`
-* [Breaking] Rename config key from `modules.http` to `modules.uhttp` to match the module name
+* [Breaking] Rename config key from `modules.http` to `modules.uhttp` to match
```

Comment:
Somebody isn't running overcommit, causing commit failures on the changelog (max line length). I would like to move this to a separate PR, but seems overkill in this case.